app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -701,6 +703,18 @@ private void saveIncompleteFormToDisk() {
         saveDataToDisk(FormEntryConstants.EXIT, false, null, true);
     }
 
+    protected void updateFormMediaToDisk() {
```

Comment:
I think the name is a bit confusing. Can I suggest calling it something like `onFormMediaUpdated` or `onExternalAttachmentUpdated`? 

What the method is doing is somewhat clear from the code, but the context about why we are doing it might be confusing to the next programmer, so this might be a good place to document why the method is called and what should be done.